packages/core/contracts/financial-templates/test/FundingRateApplierTest.sol

```diff
@@ -10,9 +10,9 @@ contract FundingRateApplierTest is FundingRateApplier {
     constructor(
         FixedPoint.Unsigned memory _initialFundingRate,
         address _fpFinderAddress,
-        address _timerAddress,
```

Comment:
changed order of params to put the timer last, as is the convention in other contracts.